bundles/osgi/pom.xml

```diff
@@ -69,6 +69,9 @@
                             <!-- Probe development tool -->
                             org.jboss.weld.probe;
 
+                            <!-- Weld Lite Extension Translator module -->
+                            org.jboss.weld.lite.extension.*;
```

Comment:
Any specific reason why `org.jboss.weld.lite.extension.*` and not `org.jboss.weld.lite.extension.translator.*`?